examples/webgl_postprocessing.html

```diff
@@ -16,7 +16,8 @@
 			{
 				"imports": {
 					"three": "../build/three.module.js",
-					"three/addons/": "./jsm/"
+					"three/addons/": "./jsm/",
+					"postprocessing": "https://esm.sh/postprocessing"
```

Comment:
This bundles its own copy of three.js. Maybe use another CDN which isn't so helpful like unpkg.com?

Reply:
With unpkg, I did not manage to get it work somehow. It searches the three dependency unfortunately. I'm no expert in importmap, if you have a quick fix, I would be glad to change it

![20230613_200718.jpg](https://github.com/mrdoob/three.js/assets/18304870/71e70559-08fd-4964-a5c2-1a591e2dc215)

![20230613_200722.jpg](https://github.com/mrdoob/three.js/assets/18304870/a5fa8712-cb1c-48db-9fb8-119362c79841)

Thank you in advance !

Reply:
I think you may have to point unpkg to the ESM file explicitly, otherwise it's serving CJS. For example:

```
"postprocessing": "https://unpkg.com/postprocessing@6.33.0/build/index.js"
```